helm/2-chart/charts/scorpio/templates/gateway-deployment.yaml

```diff
@@ -21,6 +21,10 @@ spec:
       labels:
         service: {{ .Values.gateway.name }}
     spec:
+      initContainers:
+      - name: init-myservice
+        image: busybox:1.28
+        command: ['sh', '-c', "until wget -qO - http://eureka:8761/actuator/health| grep UP; do echo hello; sleep 1; done"]
```

Comment:
replace "echo hello" by "echo waiting for eureka service"

Reply:
Done!